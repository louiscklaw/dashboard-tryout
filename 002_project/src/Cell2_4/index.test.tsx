import { render, screen } from '@testing-library/react';
import { Cell2_4 } from '.';

test('renders learn react link', () => {
  render(<Cell2_4 />);
  const linkElement = screen.getByText(/cell/i);
  expect(linkElement).toBeInTheDocument();
});

test('has clear button after typing', async () => {
  const { container } = render(<Cell2_4 />);
  expect(container).toMatchSnapshot();
});
