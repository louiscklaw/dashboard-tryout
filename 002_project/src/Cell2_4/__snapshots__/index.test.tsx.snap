// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has clear button after typing 1`] = `
<div>
  <div
    class="cell"
  >
    <div
      class="cell_title"
    >
      cell2.4 (hksingleparty-demo)
    </div>
    <div
      class="cellContent"
    >
      <div>
        <a
          href="//pa_admin.louislabs.com/"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt=""
            src="//status.iamon99.com/api/badge/111/status?label=pa_admin.louislabs.com"
          />
        </a>
      </div>
      <div>
        <a
          href="//pa_mobile.louislabs.com/"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt=""
            src="//status.iamon99.com/api/badge/112/status?label=pa_mobile.louislabs.com"
          />
        </a>
      </div>
      <div>
        <a
          href="//pa_backend.louislabs.com/"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt=""
            src="//status.iamon99.com/api/badge/113/status?label=pa_backend.louislabs.com"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;
